journal-writings/original/06-chapter-4.adoc

```diff
@@ -485,7 +490,11 @@ you may find true peace and satisfaction.
 If the Lord be our God, in truth and reality, there is safety for us;
 for he is a strong hold in the day of trouble, and knoweth them that trust in him.
 
+[.signed-section-context-close]
 Isle of Wight county, in Virginia, 29th of the Fifth month, 1757.
+//This also different than the modernized...one of the lines near the end is in a different  paragraph. Also, in the original PDF, his name isn't there so I left as it is in the original.
```

Comment:
ok, i'm not sure... why don't you pull this comment and I've made a note to go back and research which is correct and decide how to fix it.  so leave it as is for now, and i'm going to do a quick pass on this after I merge, thanks.